core/src/test/scala/unit/kafka/server/ServerGenerateBrokerIdTest.scala

```diff
@@ -187,4 +187,12 @@ class ServerGenerateBrokerIdTest extends ZooKeeperTestHarness {
     }
     true
   }
+
+  @Test
+  def testGetSequenceIdMethod {
```

Comment:
We usually have `()` for test methods (because they generally include side-effects). Not my choice, but it is what we do.